dcos/http/client/client_test.go

```diff
@@ -0,0 +1,76 @@
+package client
+
+import (
+	"crypto/x509"
+	"net/http"
+	"reflect"
+	"testing"
+)
+
+const (
+	CACertPath     = "fixtures/root_ca_cert.pem"
+	ServiceAccount = "fixtures/test_service_account.json"
+)
+
+func TestLoadCAPool(t *testing.T) {
+	caPool, err := loadCAPool(CACertPath)
+
+	if err != nil {
+		t.Error("Expected no errors loading CA cert fixutre, got", err)
+	}
+
+	if reflect.TypeOf(caPool) != reflect.TypeOf(&x509.CertPool{}) {
+		t.Errorf("loadCAPool() returned invalid type, got %T", caPool)
+	}
+
+	_, defErr := loadCAPool("fake/cert/path")
+	if defErr == nil {
+		t.Error("Expected error with bad path, got", defErr)
+	}
+}
+
+func TestGetTransport(t *testing.T) {
+	transport, err := getTransport(CACertPath)
+
+	if err != nil {
+		t.Error("Expected nil error getting transport, got", err.Error())
+	}
+
+	if reflect.TypeOf(transport) != reflect.TypeOf(&http.Transport{}) {
```

Comment:
not sure how useful this as a "test". the interface for `getTransport` declares that it returns `*http.Transport` -- why bother testing it? it can't return anything else...